app/models/game.rb

```diff
@@ -20,14 +20,23 @@ def add_player(player)
 
   def set_up_game
     Message.destroy_all
-    update(ordered_players: players.shuffle.map do |player|
-      player.class == AiPlayer ? "a" + player.id.to_s : player.id.to_s
-    end)
+    find_order
     set_blinds
     load_deck
     deal_pocket_cards
   end
 
+  def find_order
+    if ordered_players == []
+      current_order = players.shuffle.map do |player|
+        player.class == AiPlayer ? "a" + player.id.to_s : player.id.to_s
```

Comment:
Very minor, but you could use `player.is_a?(AiPlayer)` rather than the explicit class check. This potentially helps subclasses in the future